gas-estimation/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+mod eth_node;
+mod ethgasstation;
+mod gasnow;
+mod gnosis_safe;
+mod linear_interpolation;
+mod priority;
+
+pub use ethgasstation::EthGasStation;
+pub use gasnow::GasNowGasStation;
+pub use gnosis_safe::GnosisSafeGasStation;
+pub use priority::PriorityGasPriceEstimating;
+
+use anyhow::Result;
+use serde::de::DeserializeOwned;
+use std::time::Duration;
+
+pub const DEFAULT_GAS_LIMIT: f64 = 21000.0;
+pub const DEFAULT_TIME_LIMIT: Duration = Duration::from_secs(30);
+
+#[cfg_attr(test, mockall::automock)]
```

Comment:
```suggestion
```
Can remove this and the dependency to mockall in Cargo.toml .

Reply:
I think it's still needed to the Priority Gas Price estimator